lib/ui/screen-components/un-protected/TermsConditions/OnePara/index.tsx

```
@@ -0,0 +1,31 @@
+import ListItem from '../ListItem'
+
+const OnePara = ({Para}) => {
+  return (
+    <div className='my-14'>
+        <div className='flex gap-3 items-center font-bold'>
+        <h3>{Para.number}</h3>
+        <Heading heading={Para.head}/>
+        </div>
+        {Para?.paras?.map((paragraph)=>
+        { 
+           return <Paragraph paragraph={paragraph} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)